querybook/server/lib/ai_assistant/assistants/openai_assistant.py

```diff
@@ -146,13 +146,11 @@ def _generate_title_from_query(
         messages = self.title_generation_prompt_template.format_prompt(
             query=query
         ).to_messages()
-        chat = ChatOpenAI(
+        result = completion(
```

Comment:
`model`would be passed in here: https://litellm.readthedocs.io/en/latest/supported/